sklearn/ensemble/tests/test_forest.py

```diff
@@ -371,6 +376,86 @@ def test_unfitted_feature_importances(name):
         getattr(FOREST_ESTIMATORS[name](), 'feature_importances_')
 
 
+@pytest.mark.parametrize("name", FOREST_CLASSIFIERS_REGRESSORS)
+@pytest.mark.parametrize("dtype", (np.float64, np.float32))
+def test_oob_importances(name, dtype):
+
+    X = X_large.astype(dtype=dtype, copy=False)
+    y = y_large.astype(dtype=dtype, copy=False)
+
+    ForestEstimator = FOREST_ESTIMATORS[name]
+
+    clf = ForestEstimator(
+        n_estimators=10,
+        random_state=0,
+        feature_importances_type="permutation",
+        bootstrap=True,
+    )
+    clf.fit(X, y)
+    importances = clf.feature_importances_
+    if name in FOREST_CLASSIFIERS:
+        imp_level = 0.025
+    else:
+        imp_level = 0.1
+
+    n_important = np.sum(importances > imp_level)
+    assert importances.shape[0] == 10
+    assert n_important == 3
+    assert np.all(importances[:3] > imp_level)
+
+
+@pytest.mark.parametrize("name", FOREST_CLASSIFIERS_REGRESSORS)
+@pytest.mark.parametrize("dtype", (np.float64, np.float32))
+def test_oob_importances_multi_class(name, dtype):
+
```

Comment:
Put a comment to explain what you are testing and why the assertion should be this way.